samcli/lib/docs/documentation.py

```diff
@@ -1,17 +1,30 @@
 """
 Library housing the logic for handling AWS SAM CLI documentation pages
 """
+import json
 import logging
+from pathlib import Path
+from typing import Any, Dict
 
 from samcli.lib.docs.browser_configuration import BrowserConfiguration
 
 LOG = logging.getLogger(__name__)
 
+DOCS_CONFIG_FILE = "documentation_links.json"
+LANDING_PAGE = "https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html"
+
 
 class Documentation:
-    def __init__(self, browser: BrowserConfiguration, url: str):
+    def __init__(self, browser: BrowserConfiguration, command: str):
         self.browser = browser
-        self.url = url
+        self.command = command
```

Comment:
nit: Doc string for the arguments

Reply:
I don't think we typically add docstrings for constructors but I don't see why not. I'll add it in!

Reply:
Yeah I see that in some places we do, but then in other places there isn't a docstring but it is nice to have one